#  * \param  c0  the first column of the matrix.
#  * \param  c1  the second column of the matrix.
#  * \param  c2  the third column of the matrix.

Vector3 c0
Vector3 c1
Vector3 c2


